app/assess/helpers.py

```diff
@@ -77,3 +84,48 @@ def resolve_application(
         referrer=request.referrer,
         display_status=display_status,
     )
+
+
+def extract_questions_and_answers_from_json_blob(
+    application_json_blob,
+) -> dict:
+    """function takes the form data and returns
+    dict of questions & answers.
+    """
+    questions_answers = defaultdict(dict)
+    forms = application_json_blob["forms"]
+
+    for form in forms:
+        form_name = form["name"]
+        for question in form[NotifyConstants.APPLICATION_QUESTIONS_FIELD]:
+            for field in question["fields"]:
+                question_title = field["title"]
+                answer = field.get("answer")
+                if field["type"] == "file":
+                    # we check if the question type is "file"
+                    # then we remove the aws
+                    # key attached to the answer
+
+                    if isinstance(answer, str):
+                        answer = answer.split("/")[-1]
+                elif (
+                    # if it's a bool we display yes/no instead of true/false
+                    isinstance(answer, bool)
+                    and field["type"] == "list"
+                ):
+                    answer = "Yes" if answer else "No"
+                questions_answers[form_name][question_title] = answer
+    return questions_answers
+
+
+def generate_text_of_application(q_and_a: dict):
+    output = StringIO()
+
+    output.write(f"********* {Config.COF_FUND_NAME} **********\n")
```

Comment:
Can we add a TODO here? we're going to need to fetch the fund info and make this dynamic once we're running multiple funds.

Reply:
done 👍🏾